stylize/__main__.py

```diff
@@ -57,6 +57,9 @@ def main():
 
     # register any formatter-specific arguments
     for formatter in formatters:
+        if formatter.get_command() == "false":
+            print("[ERR] A required dependency was not found. Check to see if clang-format is available on your path.")
```

Comment:
Actually want to make the default return None and then implement get_command() for the YapfFormatter to be consistent.  The one for Yapf would probably just be `return shutil.which('yapf')`.  And yeah that's fine, squashes are cool.